public/app/features/explore/QueryField.tsx

```diff
@@ -622,9 +623,33 @@ export class QueryField extends React.PureComponent<QueryFieldProps, QueryFieldS
 
   handleCopy = (event: ClipboardEvent, change: Editor) => {
     event.preventDefault();
-    const selectedBlocks = change.value.document.getBlocksAtRange(change.value.selection);
-    event.clipboardData.setData('Text', selectedBlocks.map((block: Block) => block.text).join('\n'));
+    const selectedBlocks: List<Block> = change.value.document.getBlocksAtRange(change.value.selection);
```

Comment:
Can we have a test for the meat of the function, i.e., the part that does not use `event`? Might have to extract that part.

Reply:
Sure, no problem